test/server/mocks/repositories/StatisticsRepository.ts

```diff
@@ -2,19 +2,12 @@
 
 import { injectable } from 'inversify'
 import { StatisticsRepositoryInterface } from '../../../../src/server/repositories/interfaces/StatisticsRepositoryInterface'
+import { GlobalStatistics } from '../../../../src/server/repositories/types'
 
 @injectable()
 export class MockStatisticsRepository implements StatisticsRepositoryInterface {
```

Comment:
Surely you could use `jest.mock` for this..?

Reply:
As in to mock to repository?

Reply:
@LoneRifle i think this is fine though. one significant benefit of using DI is to easily swap out implementation when necessary, including tests.

Reply:
Not disputing that - I'm saying that instead of creating a specific class for testing, you just supply one dynamically created using mocks